src/viam/app/app_client.py

```diff
@@ -8,10 +8,9 @@
 
 from viam import logging
 from viam.app._logs import _LogsStream, _LogsStreamWithIterator
+from viam.proto.app import AddRoleRequest, APIKeyWithAuthorizations, AppServiceStub
 from viam.proto.app import (
-    AddRoleRequest,
-    APIKeyWithAuthorizations,
-    AppServiceStub,
+    AuthenticatorInfo,
```

Comment:
(minor) pretty weird that these two import lines are separated out! I know the `make format` suggested we do that, but I suspect that's because there was a `import Foo as FooPB` line in there to explain why. I'd suggest making this all one import statement again.